### PR TITLE
Add result redirect unit test

### PR DESCRIPTION
**Pull request by dmahely**
This PR includes a unit test to see whether clicking on 'Continue' in the result page of the first round will redirect to the second round. I'm not sure about the last two lines, where I'm awaiting for the round component to be visible, specifically if the text 2/5 is visible, then expecting that 2/5 is visible. Should I change what I'm awaiting for to be the name of the album or one of the artists that I know will be there in the second round?

```javascript
// on result component
await screen.findByText(/Correct/i);
const continueButton = screen.getByRole('button', { name: 'Continue' });
fireEvent.click(continueButton);
// on round component
await screen.findByText('2/5');
expect(screen.getByText('2/5')).toBeVisible();
```

### CODE REVIEW

File: src/components/App/App.test.js [hunk 1]
```diff
@@ -70,3 +70,26 @@ test('selecting correct answer shows correct result', async () => {
...
+    fireEvent.click(fiveRoundsButton);
+
+    await screen.findByText('1/5');
+    const correctOption = screen.getByRole('img', {
+        name: 'Juice WRLD',
+    });
+    fireEvent.click(correctOption);
+
+    await screen.findByText(/Correct/i);
+    const continueButton = screen.getByRole('button', { name: 'Continue' });
```
Comment (tryggvigy) [inline src/components/App/App.test.js]: really minor, but you can also use regular expressions that ignore casing in the name. It doesn't really matter to a user if they see "continue" or "Continue", so maybe it shouldn't matter to the test either. Depends a little bit on context and the system being tested though. Like most other rules, it's not an absolute rule!

**Review by tryggvigy — APPROVED** [review]
LGTM!

> Should I change what I'm awaiting for to be the name of the album or one of the artists that I know will be there in the second round?

I think it's fine! It could be nice to assert that 4 options are also rendered on the second screen to make the test cover even more. You could imagine, this test would pass if "2/5" is showing but for some reason the actual options did not render properly due to some bug for example.

It's a matter of drawing the line somewhere though, it's not worth it spend a lot of time and effort to test every single detail. I'm not sure if it's necessary here anyway since we have an error boundary that handles showing an error page in case of errors, which would not include the "2/5" text

**Comment by dmahely** [comment]
> I think it's fine! It could be nice to assert that 4 options are also rendered on the second screen to make the test cover even more. You could imagine, this test would pass if "2/5" is showing but for some reason the actual options did not render properly due to some bug for example.
> 
> It's a matter of drawing the line somewhere though, it's not worth it spend a lot of time and effort to test every single detail. I'm not sure if it's necessary here anyway since we have an error boundary that handles showing an error page in case of errors, which would not include the "2/5" text

Hmm so if we test that clicking on 'continue' in the first round result will redirect to the second round which has 4 artist options, do you think it would make sense to do this for each round, at least in the case of 5 rounds?

**Comment by dmahely** [comment]
Perhaps adding a unit test to ensure that the backend always returns 4 options is a better way to go about it

**Pull request merged**